addon/package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "lint": "eslint .",
     "start": "jpm run -b /usr/bin/firefox --prefs dev-prefs.json",
+    "watch": "jpm watchpost --post-url http://localhost:8888",
```

Comment:
It's a small thing, but this should probably be split off into a separate PR because it's not directly related to the task in this PR.